app/org/sagebionetworks/bridge/models/surveys/BloodPressureConstraints.java

```diff
@@ -0,0 +1,12 @@
+package org.sagebionetworks.bridge.models.surveys;
+
+import static org.sagebionetworks.bridge.models.surveys.UIHint.BLOODPRESSURE;
+
+import java.util.EnumSet;
+
+public class BloodPressureConstraints extends Constraints {
+    public BloodPressureConstraints() {
+        setDataType(DataType.STRING);
```

Comment:
I would argue for DataType.BLOODPRESSURE along with documentation of the data being uploaded (systolic, diastolic, unit).